### PR TITLE
[Component with its own class name as service type] LPS-198943 Refactor Helper components from commerce modules part 1

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-198943](https://liferay.atlassian.net/browse/LPS-198943)

Previous Pull: https://github.com/liferay-core-infra/liferay-portal/pull/6133

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java [hunk 1]
```diff
@@ -849,10 +890,6 @@ private CPDefinition _updateNestedResources(
 			ProductConfigurationUtil.updateCPDefinitionInventory(
 				_cpDefinitionInventoryService, productConfiguration,
 				cpDefinition.getCPDefinitionId());
-
-			ProductConfigurationUtil.updateCPDAvailabilityEstimate(
```
Comment (dantewang) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:853]: @RenanUmV Why are some of the logic in this class removed? It's very hard to see the reason from the commit message, as it just says "merge xx into xx". Please separate the merge and the removal in different commits, and give detailed explanation.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java [hunk 2]
```diff
@@ -865,11 +865,15 @@ private CommerceOrder _updateOrder(CommerceOrder commerceOrder, Order order)
 					order.getPrintedNote(), commerceOrder.getPrintedNote()));
 		}
 
-		commerceOrder = _commercePaymentEngine.updateOrderPaymentStatus(
-			commerceOrder.getCommerceOrderId(),
-			GetterUtil.getInteger(
-				order.getPaymentStatus(), commerceOrder.getPaymentStatus()),
-			commerceOrder.getTransactionId(), StringPool.BLANK);
+		if ((order.getPaymentStatus() != null) &&
```
Comment (dantewang) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java:868]: @RenanUmV This change doesn't look like an SF, please separate it and explain why.

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java [hunk 3]
```diff
@@ -534,10 +534,10 @@ private Long[] _getOrderItemIds(OrderItem[] orderItems) {
 	}
 
 	private Page<Order> _getOrdersPage(
-		long companyId, Filter filter, Pagination pagination, String search,
-		Sort[] sorts,
-		UnsafeFunction<Document, Order, Exception> transformUnsafeFunction,
-		boolean useSearchResultPermissionFilter)
+			long companyId, Filter filter, Pagination pagination, String search,
```
Comment (dantewang) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java:537]: @RenanUmV This is not a big problem, but we should avoid it. When you merge the helpers, run SF before making each commit, so the merging commits won't contain unnecessary indent changes, and this won't appear in your SF commit which mainly aims to inline methods.

**Review by dantewang — CHANGES_REQUESTED** [review]
cc @rafaprax

**Comment by dantewang** [comment]
Please resend, thanks.

**Comment by RenanUmV** [comment]
Reset here: https://github.com/liferay-core-infra/liferay-portal/pull/6225

**Pull request closed without merging**